packages/jaeger-ui/src/components/TracePage/TraceStatistics/tableValues.tsx

```diff
@@ -315,18 +147,18 @@ function valueFirstDropdown(selectedTagKey: string, trace: Trace) {
     for (let j = 0; j < allSpans.length; j++) {
```

Comment:
Just to illustrate my earlier point about linear searches: with this loop in place we're doing O(N^2) work.